src/Web/Sfa.Das.Sas.Shared.Components.Web/Sfa.Das.Sas.Shared.Components.Web.csproj

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <TargetFramework>netcoreapp2.2</TargetFramework>
     <AspNetCoreHostingModel>InProcess</AspNetCoreHostingModel>
+    <UserSecretsId>21123918-648b-445a-8f71-7ca8156967d2</UserSecretsId>
```

Comment:
whats this do? is it relevant only to your environment?

Reply:
Bloody VS 👿 👍